packages/create-amplify/src/amplify_project_creator.ts

```diff
@@ -32,19 +32,23 @@ export class AmplifyProjectCreator {
     private readonly initialProjectFileGenerator: InitialProjectFileGenerator,
     private readonly npmInitializedEnsurer: NpmProjectInitializer,
     private readonly gitIgnoreInitializer: GitIgnoreInitializer,
-    private readonly projectRoot: string
+    private readonly projectRoot: string,
+    private readonly printer: Printer
```

Comment:
Since we're going with the `import { printer } from './printer'` strategy, we should be consistent here as well (ie don't inject in the ctor, just import it)